hub/parameters/digitalhubdev.parameters.json

```diff
@@ -0,0 +1,12 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "digitalhub_databaseAccount_name": {
```

Comment:
While we're still bikeshedding I wonder if it's worth making this less flexible?

If we had a single param called `environmentName` or similar then we could enforce that the internal names are all consistent bar the suffix.